src/Nethermind/Nethermind.Core.Test/Json/TxTypeConverterTests.cs

```diff
@@ -16,6 +16,7 @@ public class TxTypeConverterTests : ConverterTestBase<TxType>
         [TestCase((TxType)255)]
         [TestCase(TxType.Legacy)]
         [TestCase(TxType.AccessList)]
+        [TestCase(TxType.Blob)]
```

Comment:
Make it a TestCaseSource and have all TxType by default here + additional values?

Reply:
Made such source, looks quite OK